tests/runner/test_runner.py

```diff
@@ -42,6 +42,33 @@ def test_get_mode_command(self):
                 'exec_command': 'sb exec',
```

Comment:
Shall we improve the code coverage a little bit?

Reply:
updated